draft-ietf-jose-hpke-encrypt.md

```diff
@@ -149,38 +149,49 @@ This specification uses the following abbreviations and terms:
 - Authenticated Encryption with Associated Data (AEAD), see {{I-D.ietf-hpke-hpke}} and {{RFC7516}}.
 - Additional Authenticated Data (AAD), see {{I-D.ietf-hpke-hpke}} and {{RFC7516}}.
 
+This specification defines the following terms.  When the term is already defined in JWE {{RFC7516}}, the definition below replaces it.
+
+Key Management Mode
+: A method of determining the Content Encryption Key (CEK) value to use and whether the encryption uses a CEK. Key Management Modes employed by this specification are Key Encryption, Key Wrapping, Direct Key Agreement, Key Agreement with Key Wrapping, Direct Encryption, HPKE Integrated Encryption, and HPKE Key Encryption.
```

Comment:
```suggestion
: A method of determining the Content Encryption Key (CEK) value to use and whether the encryption uses a CEK. Key Management Modes employed by this specification are HPKE Integrated Encryption and HPKE Key Encryption.
```

Reply:
This text replaces an existing definition in JWE [RFC 7516] and so all the key management modes must continue to be listed.